two-libs/lib/build.gradle

```diff
@@ -0,0 +1,43 @@
+apply plugin: "com.android.model.native"
+
+model {
+    android {
+        compileSdkVersion = 23
+        buildToolsVersion = "23.0.0"
```

Comment:
Only the compileSdkVersion is needed.  Anything else in the android{} block should be removed.